modules/mobile/app/controllers/mobile/v0/maintenance_windows_controller.rb

```diff
@@ -28,7 +28,8 @@ class MaintenanceWindowsController < ApplicationController
         %i[mhv_sm secure_messaging],
         %i[mhv_meds rx_refill],
         %i[vaos appointments],
-        %i[vet360 user_profile_update],
+        %i[vapro_personal_info user_demographics],
+        %i[vapro_contact_info user_contact_update],
```

Comment:
FE currently uses `user_profile_update` for both person info page and contact information. (https://adhoc.slack.com/archives/C0212B872MT/p1731433052180219). This will require a change on the FE side since we now have two windows for each page. I've confirmed with Tom that personal_info maps to demographics and contact_info maps to user/address, user/email, user/address update endpoints. 

This update makes it so the new windows will not be used on FE until they make the change. The alternative is we keep `%i[vet360 user_profile_update],` as well as add these new windows to avoid changing behavior until FE integrates with the change. I figured that we now know that vet360 is too broad of a window then it'd be better to have instances of users trying to hit the endpoints during a window that FE hasn't integrated with then continue using an overly broad window that blocks users from using the endpoint when the endpoint would have actually worked.

Reply:
new ticket: https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/10167